### ISSUE TITLE
downlaoding speed issues (not my connection)

### ISSUE THREAD

**Issue by iNoSec2** [ytdl-org/youtube-dl#30311]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

My youtube-dl version:
![image](https://user-images.githubusercontent.com/51358868/144554856-f72b5fb8-3e92-44a4-8d0b-93f4f7752a02.png)

## Question

Since a few days youtube-dl download very slowly (10x less), even with different user-agent, my connection works perfectly
![image](https://user-images.githubusercontent.com/51358868/144554509-5a30654a-bef5-459e-87e9-02ceb0f1f05f.png)
Is there some others users who have same issues since at least 4 days? A change in youtube code?

Thanks for people who will take the time to look for

PS: here you can see the dl speed for a youtube video (same speed for any youtube videos)
![image](https://user-images.githubusercontent.com/51358868/144555101-e9228bbc-7580-49d8-9cfc-9e56f7cefa1d.png)

**Comment by kikuyan** [comment]
Look around other open issue reports before making a new one. Too many duplicates.

**Comment by nicolaasjan** [comment]
Yeah.
Do a search first...
https://github.com/ytdl-org/youtube-dl/issues?q=slow+download

**Comment by iNoSec2** [comment]
And maybe you can stop being a parrot asshole 😉

**Comment by dirkf** [comment]
TL;DR:
Duplicate of #29326, fixed in PR #30184.

**Comment by 89z** [comment]
@iNoSec2 youre the asshole. Search for speed shows at least 15 open issues:

https://github.com/ytdl-org/youtube-dl/search?q=speed+in%3Atitle&state=open&type=issues
